src/style.css

```
@@ -1,3 +1,76 @@
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  justify-content: space-between;
+  width: 100vw;
+  align-items: center;
+}
+
+.logo {
+  cursor: pointer;
+  width: 40px;
+  height: 40px;
+  padding: 10px 0;
+  margin-left: 50px;
+}
+
+.nav__link {
+  display: flex;
+  white-space: nowrap;
+  height: 60px;
+  margin: 0 auto;
+}
+
+.nav__item {
+  display: flex;
+  align-items: center;
+  padding: 10 20px;
```

```suggestion
  padding: 10 20px;
```

10 what? Did you forget about px?